share/picongpu/tests/compileAtomicPhysics/include/picongpu/param/atomicPhysics.param

```diff
@@ -0,0 +1,101 @@
+#include "picongpu/particles/atomicPhysics/stateRepresentation/ConfigNumber.hpp"
+
+#include <cstdint>
+
+namespace picongpu::atomicPhysics
+{
+    // configuration of electron ElectronHistogram
+    //{
+    struct MaxEnergyParam
+    {
+        // eV
+        static constexpr float_X maxEnergy = float_X(1e7);
+    };
+    using ElectronHistogram = particles::atomicPhysics::electronDistribution::LogSpaceHistogram<
+        100u, // T_numberBins
+        MaxEnergyParam>;
+    //}
+
+    // atomicPhysics rate solver settings
+    //{
+    //      atomicPhysics timeStepLength sub-stepping of numerical limit
+    constexpr float_X timeStepAlpha = 0.3_X;
+
+    //      which probability approximation to use for the acceptance step
+    using ProbabilityApproximationFunctor = picongpu::particles::atomicPhysics::ExponentialApproximationProbability;
+    //}
+
+    /** atomicConfigNumber definition for argon ions
+     *
+     * @details describes analytic bidirectional indexation of atomic states, i.e. conversion of configNumber to/from
+     *  occupation number vector
+     *
+     * define one type-instance for every physical ion species, i.e. Ar, Cu, ...
+     *
+     * See piconpgu/particles/atomicPhysics/stateRepresentation/ConfigNumber.hpp for further information.
+     */
+    using ConfigNumber_Argon = particles::atomicPhysics::stateRepresentation::ConfigNumber<
+        /** data type to use for configNumber representation
+         * @attention must be able to hold all configNumbers in atomic data input files! */
+        uint64_t,
+        /** maximum(inclusive) principial quantum number n_max to include in modelling
+         *
+         * corresponds to number of atomic shells to include in indexing of atomic states.
+         *
+         * @attention must be <= to number of shells used in atomic data input
+         * @attention only influences analytic indexation, actually tracked atomic states are specified via atomic
+         *  input data files of each atomicPhyscis ion species, @note see speciesDefinition.param
+         */
+        10u,
+        //! atomic number of element, @attention must be < 256!
+        18u>;
+
+    /** atomicData dataBase type
+     *
+     * define one type-instance for every physical ion species, i.e. Ar, Cu, ...
+     *
+     * see picongpu/particles/atomicPhysics/atomicData/AtomicData.hpp for further information.
+     */
+    using AtomicData_Ar = particles::atomicPhysics::atomicData::AtomicData<
+        uint32_t, // T_Number
+        float_X, // T_Value
+        uint32_t, // T_CollectionIndex
+        ConfigNumber_Argon, // T_ConfigNumber, data type used for storage
+        float_64, // T_Multiplicities, data type used for storage
+        true, // T_electronicExcitation
+        true, // T_electronicDeexcitation
+        true, // T_spontaneousDeexcitation
+        true, // T_electronicIonization
+        true, // T_autonomousIonization
+        false>; // T_fieldIonization
```

Comment:
REFACTOR: Could these be enums such that we say `ElectronicExcitation::On` and so on?